test/Makefile

```diff
@@ -0,0 +1,16 @@
+CXX ?= c++
```

Comment:
It'd be good to update Travis and appveyor configuration to run these tests in CI also

Reply:
Done (for AUTOTOOLS=no only)